package.json

```diff
@@ -51,6 +51,7 @@
     "react-icons": "^3.0.0",
     "react-letter": "^0.3.1",
     "react-markdown": "^8.0.3",
+    "react-qr-code": "^2.0.8",
```

Comment:
We already have a qr code library `qrcode.react` that you can use, so we don't need two libraries.